docs/dev/building-on-zksync/hello-world.md

```diff
@@ -64,6 +65,7 @@ module.exports = {
       url: "https://testnet.era.zksync.dev",
       ethNetwork: "goerli", // RPC URL of the network (e.g. `https://goerli.infura.io/v3/<API_KEY>`)
       zksync: true,
+      verifyURL: 'https://zksync2-testnet-explorer.zksync.dev/contract_verification'  // Verification endpoint for Goerli
```

Comment:
wondering about this url?

Reply:
The comment was misleading with the mention of Goerli. Updated. The URL is correct.